airframe-sql/src/main/scala/wvlet/airframe/sql/model/Expression.scala

```diff
@@ -239,12 +239,12 @@ object Expression {
   }
   case class SingleColumn(
       expr: Expression,
-      alias: Option[Expression],
+      alias: Option[String],
```

Comment:
Make sense. A concern is handling various quotation types (e.g., unquoted/quoted identifier) in SQLGenerator.

Reply:
For clarity, let's mention that alias can be unquoted/quoted strings.